repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -55,6 +57,14 @@ class ExecutorClassLoader(conf: SparkConf, classUri: String, parent: ClassLoader
     }
   }
 
+  override def getResource(name: String): URL = {
+    parentLoader.getResource(name)
```

Comment:
That all seams reasonable to me (sorry for the slow reply I'm in Tokyo right now).